eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/impl/AlertEagleStorePlugin.java

```diff
@@ -62,10 +60,9 @@ public void onAlert(AlertStreamEvent event) throws Exception {
         if (eventList == null || eventList.isEmpty()) {
             return;
         }
-        List<AlertPublishEvent> alertEvents = new ArrayList<>();
-        for (AlertStreamEvent e : eventList) {
-            alertEvents.add(AlertPublishEvent.createAlertPublishEvent(e));
-        }
+        List<AlertPublishEvent> alertEvents = eventList.stream().map(
```

Comment:
i feel its fancy to use newer java features, but old syntax is more readable, may be its just a coding preference